### ISSUE TITLE
http://sharejs.org/ is down?

### ISSUE THREAD

**Issue by drpepper** [share/ShareJS#395]
http://downforeveryoneorjustme.com/http://sharejs.org/

**Comment by netsgnut** [comment]
It seems [it was intentional](https://groups.google.com/forum/?fromgroups#!topic/sharejs/NGJ-I9Zuz_8).
>  I took sharejs.org down the other day and I felt good about it - I felt like the site was lying. (And it was still running 0.5 if you can believe it.)

**Comment by drpepper** [comment]
Thanks for the link!

It would probably be good to put something on the site, though, even just a link to the email.

**Comment by netsgnut** [comment]
I agree. Have been tracking this project for some time and the information, historical reasons I know, is often a bit undermaintained.

Hope things would get better after the ShareJS/ShareDB split.
